lib/WsRtspVideo.tsx

```diff
@@ -180,5 +183,17 @@ export const WsRtspVideo: React.FC<WsRtspVideoProps> = ({
     }
   }, [play, pipeline, fetching])
 
-  return <VideoNative autoPlay={autoPlay} muted={muted} ref={videoRef} />
+  useEffect(() => {
+    if (videoRef.current !== null && volume !== undefined) {
+      videoRef.current.volume = volume
+    }
+
+    if (volume !== undefined && volume > 0) {
+      setSilent(false)
+    } else {
+      setSilent(true)
+    }
+  }, [volume])
```

Comment:
This should be moved to the Player. You can access the video element there via the onPlaying event, check what type it is (e.g. if the element has volume) and couple it directly to the controls. The reason is that we want to avoid anything here that is not connected to delivering the video stream.

Reply:
Alright, will look in to that. Like I said, I wasn't sure how to implement this properly. :see_no_evil: